product/product.go

```diff
@@ -16,18 +16,15 @@ const (
 	Vault  = "vault"
 )
 
-const (
```

Comment:
Heh, well, about that earlier comment about putting these into the `product` package... 😄 Maybe we SHOULD go ahead and do that?

Reply:
Yeah - honestly it makes the most sense to keep the consts in the product pkg. I wondered a bit about how to make it available via the agent to keep the dependency graph cleaner... but this feels like the way to go.